tests/GenericSparqlTest.php

```diff
@@ -1121,12 +1121,33 @@ public function testListConceptGroupContentsIncludingDeprecatedConcept()
    * @covers GenericSparql::generateChangeListQuery
    * @covers GenericSparql::transFormChangeListResults
    */
-  public function testQueryChangeList()
+  public function testQueryChangeListCreatedNoDeprecated()
   {
     $voc = $this->model->getVocabulary('changes');
     $graph = $voc->getGraph();
+    $sparql = new GenericSparql('http://localhost:13030/skosmos-test/sparql', $graph, $this->model);
```

Comment:
SPARQL endpoint address should be set via a variable, see #1166 for more information.